Makefile

```diff
@@ -122,7 +124,7 @@ generate: build-local embedmd
 	@$(GOPATH)/bin/embedmd -w `find . -path ./vendor -prune -o -name "*.md" -print`
 
 embedmd:
-	go get github.com/campoy/embedmd@v1.0.0
+	GO111MODULE=off go get github.com/campoy/embedmd
```

Comment:
Yup I was going to suggest this. This `go mod` limitation is very annoying. You can see what the prometheus team had to do in order to get around this issue:

https://github.com/prometheus/prometheus/blob/a5d39361ab0b53081fef5bdc6c722148716f175d/Makefile.common#L187-L201

Reply:
^ That's just an FYI btw. I am definitely okay with go getting the embedmd without the versionedPath. Hopefully it works :)

Reply:
That looks interesting 😄

I removed the version restrain, test pass now, tbh the embedmd project does not change much. But up to you?

Reply:
Fine by me.